scripts/ubuntu/mongo-orchestration.sh

```diff
@@ -23,11 +23,11 @@ dpkg -x mongodb-enterprise-server_3.2.0_amd64.deb 3.2.0
 dpkg -x mongodb-enterprise-mongos_3.2.0_amd64.deb 3.2.0
 dpkg -x mongodb-enterprise-server_3.6.0_amd64.deb 3.6.0
 
-
 # Python stuff for mongo-orchestration
-apt-get install -y python python-dev python-pip
+apt-get install -y python python-dev
+python get-pip.py
```

Comment:
This is the install process suggested in the [pip documentation](https://pip.pypa.io/en/stable/installing/).